Resources/views/Template/attribute.form.html.twig

```diff
@@ -11,6 +11,15 @@
         </div>
     </div>
```

Comment:
please adapt template to look like categories:
![image](https://cloud.githubusercontent.com/assets/5152633/16679013/c330a3fc-44e4-11e6-90fc-54fb7b6a8c5e.png)